src/main/scala/io/iohk/ethereum/consensus/validators/BlockHeaderValidatorSkeleton.scala

```diff
@@ -77,7 +78,7 @@ abstract class BlockHeaderValidatorSkeleton(blockchainConfig: BlockchainConfig)
       _ <- validateGasUsed(blockHeader)
       _ <- validateGasLimit(blockHeader, parentHeader)
       _ <- validateNumber(blockHeader, parentHeader)
-      _ <- validateOptOut(blockHeader)
+      _ <- validateExtraFields(blockHeader)
```

Comment:
For attack purposes or for facilitating our usage of it?

I'm not sure if the order of the validations should matter that much here as they all are quite simple and easy to check (except from the PoW validation that probably takes more). So I'm not convinced if it would be better for them to have a particular order